duo-client/src/main/java/com/duosecurity/client/Http.java

```diff
@@ -139,9 +140,16 @@ public String executeRequestRaw() throws Exception {
   public Response executeHttpRequest() throws Exception {
     String url = "https://" + host + uri;
     String queryString = canonQueryString();
+    String jsonBody = canonJSONBody();
     RequestBody requestBody;
     if (sigVersion == 1 | sigVersion == 2){
```

Comment:
I didn't catch this before but the OR should be double pipes ||
One pipe technically works because booleans, but let's be safe.